sycl/test-e2e/Reduction/reduction_reducer_op_eq.cpp

```diff
@@ -138,7 +138,13 @@ int test(queue &Q, T Identity) {
 
   int Error = 0;
   if constexpr (IsFP) {
-    T Diff = (Expected / *Res) - T{1};
+    T Diff;
+    Diff.x() = (std::abs(Res->x()) < 1e-6 && std::abs(Expected.x()) < 1e-6)
```

Comment:
Why is it `1e-6` and not some other number?

Reply:
Because the float type can only be accurate to the last six digits